modules/ztp-precaching-downloading-artifacts.adoc

```diff
@@ -215,7 +215,7 @@ You need to include the {rh-rhacm} hub and MCE Operator versions by using the `-
 <1> Specifies the downloading function of the {factory-prestaging-tool}.
 <2> Defines the {product-title} release version.
 <3> Defines the {rh-rhacm} version.
-<4> Defines the MCE version.
+<4> Defines the {mce-short} version.
```

Comment:
Again, is "multicluster engine operator version" the same as "multicluster engine version"?